docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+version: "3"
+services:
+  express-server:
+    build: .
+    image: ${DOCKERHUB_NAME}/${IMAGE_NAME}/${IMAGE_VERSION}
+    env_file: .env
```

Comment:
`docker-compose.yml` doesn't usually have `env_file` defined because that gets set in `docker-compose.override.yml` or or injected at runtime on deployment.